src/containers/Card.js

```diff
@@ -122,7 +131,7 @@ Card.propTypes = {
   fontFamily: PropTypes.string,
 };
 
-const styles = StyleSheet.create({
+export const styles = StyleSheet.create({
```

Comment:
Not sure how many people will actually it especially since you need to import it like such `react-native-elements/src/containers/Card`, but if it's useful to you, I am OK with doing this.